crepecake/src/main/java/com/google/cloud/tools/crepecake/json/ImageToJsonTranslator.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.crepecake.json;
+
+import com.google.cloud.tools.crepecake.blob.Blob;
+import com.google.cloud.tools.crepecake.blob.BlobDescriptor;
+import com.google.cloud.tools.crepecake.image.DescriptorDigest;
+import com.google.cloud.tools.crepecake.image.Image;
```

Comment:
Something I just noted (not to be addressed in this PR). Is an Image always a Docker image? Can it be some other kind of image? If not, we might want to rename this class DockerImage.

Reply:
The image can be either a Docker image or an OCI image. But in the first implementation, we will only be building a Docker image. Adding OCI image support should be fairly easy by changing some constants in the JSON templates.